.golangci.yml

```diff
@@ -0,0 +1,92 @@
+# options for analysis running
+run:
+  tests: true
+  timeout: 3m
+
+
+# all available settings of specific linters
+linters-settings:
+  dupl:
+    threshold: 150
+  funlen:
+    lines: 100
+    statements: 50
+  goconst:
+    min-len: 2
+    min-occurrences: 3
+  gocritic:
+    enabled-tags:
+      - diagnostic
+      - experimental
+      - opinionated
+      - performance
+      - style
+    disabled-checks:
+      - whyNoLint
+  gocyclo:
+    min-complexity: 15
+  goimports:
+    local-prefixes: github.com/ewohltman/ephemeral-roles
```

Comment:
Why is this needed?

Reply:
good question, copy paste from another project... will take a look at it